lib/moab/storage_object_validator.rb

```diff
@@ -92,14 +95,32 @@ def check_correctly_formed_moabs
         errors.concat check_sub_dirs(version_sub_dirs, version_dir, EXPECTED_VERSION_SUB_DIRS)
         after_result_size = errors.size
         # run the following checks if this version dir passes check_sub_dirs, even if some prior version dirs didn't
+        errors.concat check_required_manifest_files(version_path, version_dir) if before_result_size == after_result_size
         if before_result_size == after_result_size
```

Comment:
now this has to be errors.size, not after_result_size, as the previous line can change errors.size  Probably you should just always do errors.size if you can.  :-P

Reply:
👍